### PR TITLE
Updated tests and extracted some functions

### PR DESCRIPTION
**Pull request by holosiek**
New tests covering most cases of bot responses

### CODE REVIEW

**Comment by skyman503** [comment]
Please fix failing test

**Comment by Jorenar** [comment]
> Please fix failing test

This PR adds more tests, the code failing them is already in the repository.

**Comment by skyman503** [comment]
> > Please fix failing test
> 
> This PR adds more tests, the code failing them is already in the repository.

So, please also fix that one

**Comment by Jorenar** [comment]
> > This PR adds more tests, the code failing them is already in the repository.
> 
> So, please also fix that one

That's your job ;)

**Comment by skyman503** [comment]
> > > This PR adds more tests, the code failing them is already in the repository.
> > 
> > 
> > So, please also fix that one
> 
> That's your job ;)

That's Your Uni project :)

**Comment by Jorenar** [comment]
> That's Your Uni project :)

Very funny, very.

Merge it and fix your codebase!

**Comment by Kijek3** [comment]
No, that's not "our job". Tests should always pass, as there are really simple ones. Please be aware of that or we just don't merge your changes.

**Comment by skyman503** [comment]
Your changes does not meet PEP8 standard, please fix

**Comment by Kijek3** [comment]
Also your tests doesn't even start! That's just can't be accepted in PR with "updated test" (sic!)

File: source/commands/google.py [hunk 1]
```diff
@@ -2,17 +2,19 @@
 
 from urllib.parse import quote_plus
 
+def generate_query(query: str):
```
Comment (skyman503) [inline source/commands/google.py:5]: why global? should be in google class

File: tests/zdalne_response_tests.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from source.response_utils.get_text import get_text
+
+ZDALNE_ANSWER = "Nie ma żadnych zdalnych, zdalne wymyśliliście sobie Wy, studenci."
```
Comment (skyman503) [inline tests/zdalne_response_tests.py:3]: why polish word as a variable name?

**Comment by janumiko** [comment]
In my opinion there's no need for overiding default configuration of pytest in our project. It would add unecessary complexity to the codebase, which would result in higher entry bar for new developers and members of our Ok-Bot community.  Also probably it would require an update of our CI pipeline, which wouldn't be necessary in case of situation where we would stick to default configuration. If it would be necessary we could change it in the future, but I don't see that necessity in this PR.

File: tests/pytest.ini [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[pytest]
```
Comment (skyman503) [inline tests/pytest.ini:1]: unnecessary

File: tests/pytest.ini [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+python_files = *_tests.py
+python_classes = Check
+python_functions = *_test
```
Comment (skyman503) [inline tests/pytest.ini:4]: unnecessary

**Review by skyman503 — CHANGES_REQUESTED** [review]

File: source/commands/google.py (hunk 1 above)
Comment (holosiek) [reply source/commands/google.py:5]: cant access member of non-static class

File: tests/zdalne_response_tests.py (hunk 2 above)
Comment (holosiek) [reply tests/zdalne_response_tests.py:3]: this is function name, not polish verb in this case

**Comment by koostosh** [comment]
REJECTED.
Authors show no initiative to fix their own bugs and try to impose their own rules to our team.
Tests fail because you setup them wrong, I recommend reading pytest reference manual.
Additionally there is no reason to move methods outside classes except you don't know how to write proper tests.

**Pull request closed without merging**